posts/2017-10-xx-serverless-vs-containers.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Serverless (FaaS) vs. Containers (CaaS) - when to pick which?
+description: Docker, Kubernetes, Serverless? Let's discuss the different technologies and discover their up- and downsides. 
+date: 2017-10-02
+layout: Post
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/serverless-vs-containers.png
+authors:
+  - PhilippMuns
+---
+
+Contrary to popular thought, FaaS and CaaS have some pretty important things in common.
+
+You want a modern, future-proof architecture? They both have it. You want to build that slick architecture while also leveraging the latest innovations in distributed systems and large-scale application development? Yep, they both have that too.
+
+It makes it hard to decide which one is best for *you*. But friend, you deserve to know. So we're taking off the gloves and laying it all on the line.
+
+What are the commonalities and distinctions? What are the advantages and disadvantages of each?
+
+It's serverless computing vs containerization, right now. Read on.
+
+## How did we get here?
+
+Before we jump right into the details, let’s cover some very important history.
+
+[INSERT IMAGE OF THE EVOLUTION HERE?!]
+
+### 1. Physical servers
+
+We used to build our own infrastructure in the form of physical servers. We set up those machines, deployed our code on them, scaled them and maintained them. The whole thing was a manual process, and pretty slow to boot.
+
+### 2. Server clusters and VMs
+
+Using a single physical server for one application was a waste of resources. So, we evolved our infrastructure thinking and combined multiple physical servers into a cluster.
+
+We used those so-called 'virtual machines' to run multiple applications in isolation on top of this infrastructure. Deployment and management got way faster and easier. However, server administration was still necessary and largely very manual.
+
+### 3. Entering the cloud (IaaS)
+
+Setting up and operating your own datacenter came with new operational challenges; cloud computing began to tackle those issues.
+
+Why not rent your servers and operational services individually, for a monthly fee? This approach made it way easier to scale up or down, and let teams move faster.
+
+### 4. PaaS, CaaS, FaaS, XaaS
```

Comment:
should CaaS be here?

Reply:
Good question. I added it because of containers. Maybe we can just keep `PaaS` and `FaaS`